syntax = "proto3"; 


service StatManager{ 
    rpc getSensors(Empty) returns (stream Sensor);
    rpc getMean(MeanRequest) returns (StringMessage);
}


message Empty {

}

message Sensor {
    int64 _id = 1;
    string data_type = 2;
}

message MeanRequest {
    int64 sensor_id = 1; 
    string data_type = 2;
}

message StringMessage {
    string value = 1;
}